From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 18 Jul 2019 19:26:43 +0200
Subject: latexmk

---
 Makefile.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 7dd98e9..a2fc116 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -43,13 +43,13 @@ nbib.c nbibtex.c: nbib.nw
 nbib.tex: nbib.nw
 	noweave nbib.nw | tail -n +2 | sed 's/<VERSION>/$(VERSION)/' > $@
 
-## rubber f = fixedpoint latex f
+## latexmk f = fixedpoint latex f
 
 nbib.pdf: nbib.tex
-	rubber --warn=all --verbose --pdf $<
+	latexmk --pdf $<
 
 nbib.ps: nbib.tex
-	rubber --warn=all --verbose --ps $<
+	latexmk --ps $<
 
 ## NOTES
 
