From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 18 Jul 2019 20:30:07 +0200
Subject: generate PDF

---
 Makefile.in | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index a2fc116..c26b0d9 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -10,8 +10,7 @@ OBJS=nbib.o nbibtex.o boyer-moore.o
 CPPDEFS=-DSHARE='"${datasubdir}"'
 
 all: nbibtex
-# all: nbib.pdf
-all: nbib.ps
+all: nbib.pdf
 
 nbibtex: $(OBJS)
 
@@ -33,7 +32,7 @@ clean:
 
 distclean: clean
 	rm -f Makefile bibtex.lua natbib.nbs nbibtex.1
-	rm -f nbib.c nbibtex.c nbib.tex nbib.ps nbib.pdf
+	rm -f nbib.c nbibtex.c nbib.tex nbib.pdf
 
 .PHONY: all install uninstall clean distclean
 
@@ -48,9 +47,6 @@ nbib.tex: nbib.nw
 nbib.pdf: nbib.tex
 	latexmk --pdf $<
 
-nbib.ps: nbib.tex
-	latexmk --ps $<
-
 ## NOTES
 
 # Makefile bibtex.lua natbib.nbs nbibtex.1: Makefile.in bibtex.lua.in natbib.nbs.in nbibtex.1.in
